### PR TITLE
chore: configure rollup to build multiple outputs

### PR DESCRIPTION
**Pull request by emmenko**
Instead of having one build command for each output format, we can include them in rollup.

### CODE REVIEW

**Comment by emmenko** [comment]
Sure, sorry for the poor description.

The main reason is mostly to keep the setup simple and consistent throughout the packages. Also, since we run `yarn build` each time we install/update the deps, I wanted to make the process faster and I feel like this approach does improve things.

> I am not sure I should be reviewing this.

Yeah, I just added you since you are also familiar with the rollup setup. I'll also consider if we should use the same in appkit...

**Comment by emmenko** [comment]
**Before**

```
lerna success run Ran npm script 'build' in 32 packages in 43.2s:
```

**After**

```
lerna success run Ran npm script 'build' in 32 packages in 30.6s:
```

File: rollup.config.js [hunk 1]
```diff
@@ -12,37 +15,66 @@ const babelConfig = require('./babel.config')
...
 
-const config = {
-  output: {
-    sourcemap: true,
+const createConfig = cliArgs => [
+  {
+    input: cliArgs.input,
+    output: {
+      format: 'umd',
+      file: `dist/${packageName}.umd.js`,
```
Comment (tdeekens) [inline rollup.config.js]: Are we only building one UMD bundle now?

Comment (emmenko) [reply rollup.config.js]: Look below 😊
The second one is minified

Comment (tdeekens) [reply rollup.config.js]: 🤦‍♂

**Pull request merged**